### DISCUSSION TITLE
Detention v1.1 [map release]

### DISCUSSION THREAD

**Discussion by headachez** [nzp-team/nzportable#711] [Map Releases]
![detention](https://github.com/nzp-team/nzportable/assets/114017494/bbb72991-33d3-405a-be87-c772cd309d92)

You fell asleep during class... What kind of nightmare is this?

map inspired by graduation from project lazarus (a map inspired by verruckt lol)

patch notes:

1.1: fixed fog and map not being sealed

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/15139075/Detention.zip)

**Comment by Reznar705** [comment]
really cool map! love the detail of the project lazarus poster.

**Comment by RambleysBiggestFan** [comment]
this map is an accurate depiction of what its like to be in detention

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org

**Comment by Jeyphr** [comment]
Be sure to watch my review of this map (when it comes out) https://youtu.be/CINzjIEJfU8

**Reply by Jeyphr** [reply]
It's out now!

**Comment by SgtCookieBobble** [comment]
Really want to love this one, as I love the map, but, at least on the Vita, the Zombies like to get stuck near the spawn area and all over the second floor.

**Comment by nonMat** [comment]
This map could be amazing but there are a lot of small problems such as you can jump out of the map through the windows and a lot of zombies get stuck.
